extra-resources/mods/Mythos/common.cardset

```diff
@@ -199,28 +264,63 @@ card('Zombie') {
     }
 }
 
-card('Holy Man') {
+// https://upload.wikimedia.org/wikipedia/commons/4/47/Michelangelo%2C_Damned_Soul.jpg
+// Public Domain
+card('Lost Soul') {
     creature 'Common'
-    attack 0
-    health 6
-    noAttack()
-    manaCost 20
-    onEndOfTurn {
-        pick 1 atRandom (
-                { doNothing() },
-                { heal 1 on 'you' },
-                { heal 1 on { thisCard() } },
-                { heal 1 on 1 random { creature true; ownedBy 'you'; zone 'Battlefield' } },
-                { heal 1 on 2 random { creature true; ownedBy 'you'; zone 'Battlefield' } }
-        )
-    }
-    whilePresent {
-        change HEALTH by 2 withPriority 1 on {
-            creature true
-            ownedBy 'you'
-            zone 'Battlefield'
-        }
-    }
+    flavor ''
```

Comment:
I don't think `flavor ''` is necessary - I'm pretty sure it defaults to an empty string already.

Same goes for `manaCost 0`, although it might be good to be explicit about this one.